crates/libs/sys/src/Windows/Wdk/NetworkManagement/Ndis/mod.rs

```diff
@@ -3258,32 +3258,32 @@ pub struct NDIS_TCP_CONNECTION_OFFLOAD {
 #[repr(C)]
 #[derive(Clone, Copy)]
 pub struct NDIS_TCP_IP_CHECKSUM_OFFLOAD {
-    pub IPv4Transmit: NDIS_TCP_IP_CHECKSUM_OFFLOAD_1,
-    pub IPv4Receive: NDIS_TCP_IP_CHECKSUM_OFFLOAD_0,
-    pub IPv6Transmit: NDIS_TCP_IP_CHECKSUM_OFFLOAD_3,
-    pub IPv6Receive: NDIS_TCP_IP_CHECKSUM_OFFLOAD_2,
+    pub IPv4Transmit: NDIS_TCP_IP_CHECKSUM_OFFLOAD_0,
+    pub IPv4Receive: NDIS_TCP_IP_CHECKSUM_OFFLOAD_1,
+    pub IPv6Transmit: NDIS_TCP_IP_CHECKSUM_OFFLOAD_2,
+    pub IPv6Receive: NDIS_TCP_IP_CHECKSUM_OFFLOAD_3,
 }
 #[repr(C)]
 #[derive(Clone, Copy)]
-pub struct NDIS_TCP_IP_CHECKSUM_OFFLOAD_0 {
+pub struct NDIS_TCP_IP_CHECKSUM_OFFLOAD_1 {
     pub Encapsulation: u32,
     pub _bitfield: u32,
 }
 #[repr(C)]
 #[derive(Clone, Copy)]
-pub struct NDIS_TCP_IP_CHECKSUM_OFFLOAD_1 {
+pub struct NDIS_TCP_IP_CHECKSUM_OFFLOAD_0 {
     pub Encapsulation: u32,
     pub _bitfield: u32,
 }
```

Comment:
I'm concerned about the potential compatibility impact of these struct name changes. Since some structs have identical form, developers may not get compile-time errors, making it easy for this change to go unnoticed in existing codebases. Addressing this and not annoying folks may be quite tricky, hmm...

Reply:
Fortunately, the nested structs that are identical will not cause any problems. In other cases, you get a compiler error which is relatively simple to follow.